test/integration/features/basic.feature

```diff
@@ -3,14 +3,16 @@ Feature: Basic
   As a user
   I need to be able to bring up a test environment
 
-  Scenario: Basic test scenario
-    Given executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
+  Scenario: Starting Minishift
+    Given Minishift should have state "Does Not Exist"
```

Comment:
agree to that, actually I had it there but then changed it back. I think that probably all "shoulds" could be removed.